docs/translations.md

```diff
@@ -0,0 +1,461 @@
+    createOldCatalogs: false,
+    indentation: 4,
+    sort: true,
+    failOnUpdate: process.env.CI,
+    output: 'locales/$LOCALE/$NAMESPACE.json'
+};
+```
+
+Key settings:
+- **keySeparator: false** - Allows dots in translation keys
+- **namespaceSeparator: false** - Allows colons in translation keys
+- **sort: true** - Keeps keys alphabetically sorted
+- **failOnUpdate: true in CI** - Prevents CI from passing if translations need updating
+
+## Translation Validation
+
+Ghost includes a custom linter to validate translations and ensure quality.
+
+### Running the Linter
+
+```bash
+cd ghost/i18n
+yarn lint:translations
+```
+
+### Validation Rules
+
+The linter checks for several types of errors:
+
+#### 1. Invalid Interpolations
+
+Ensures curly braces are properly matched:
+
+```json
+// ❌ Error: Unclosed {
+"Welcome to {siteTitle": "Welcome to {siteTitle"
+
+// ✅ Correct
+"Welcome to {siteTitle}": "Welcome to {siteTitle}"
+```
+
+#### 2. Unused Variables
+
+Warns when a variable is defined in the source but not used in the translation:
+
+```json
+// English source
+"Welcome to {siteTitle}!": "Welcome to {siteTitle}!"
+
+// ❌ Error: Translation does not use variable "siteTitle"
+"Welcome to {siteTitle}!": "Bienvenue !"
+
+// ✅ Correct
+"Welcome to {siteTitle}!": "Bienvenue sur {siteTitle} !"
+```
+
+#### 3. Undefined Variables
+
+Errors when a translation uses a variable not present in the source:
+
+```json
+// English source
+"Welcome!": "Welcome!"
+
+// ❌ Error: Translation uses unknown variable "name"
+"Welcome!": "Bienvenue {name} !"
+
+// ✅ Correct
+"Welcome!": "Bienvenue !"
+```
+
+### Ignoring Validation Errors
+
+Sometimes validation errors are intentional (e.g., grammatical differences between languages). These can be ignored by adding them to `test/i18n-ignores.json`:
+
+```json
+{
+    "overrides": {
+        "ghost/i18n/no-unused-variables": [
+            {
+                "file": "fr/ghost.json",
+                "key": "Some translation key",
+                "comment": "French grammar doesn't require this variable"
+            }
+        ]
+    }
+}
+```
+
+You can also automatically ignore all errors (useful for bulk updates):
+
+```bash
+yarn lint:translations --fix --unsafe-ignore-all
+```
+
+## Development Workflow
+
+### Adding New Translatable Strings
+
+1. **Write code with translation calls:**
+   ```javascript
+   t('New feature title')
+   t('Welcome {name}!', {name: userName})
+   ```
+
+2. **Extract translations:**
+   ```bash
+   cd ghost/i18n
+   yarn translate
+   ```
+
+3. **Verify extraction:**
+   - Check that new keys appear in `locales/en/*.json`
+   - Verify other locales have empty values for new keys
+   - Check `locales/context.json` was updated
+
+4. **Run tests:**
+   ```bash
+   yarn test
+   ```
+
+5. **Commit changes:**
+   Include both code changes and updated translation files in your commit.
+
+### Updating Existing Translations
+
+When modifying existing translation keys:
```

Comment:
it would probably be good to offer a note here that changing an existing key with translations and running extraction will cause deletion of any strings already attached to the old key.

(We could use better tooling here, for those times the changes to the string are modest and keeping the existing translation would be better than going back to untranslated.)